grblas/base.py

```diff
@@ -22,70 +62,138 @@ def __init__(self, gb_obj, dtype):
     def __invert__(self):
         return ComplementedMask(self)
 
+    def __delitem__(self, keys):
+        if self.is_scalar:
+            raise TypeError('Indexing not supported for Scalars')
+        raise NotImplementedError('Not available until GraphBLAS v1.3')
```

Comment:
How soon can we get GraphBLAS 1.3? ;)

Reply:
That's up to Tim Davis. His 3.2.0 release was about speed optimizations for the GAP benchmark. I think he will start on 1.3 compatibility next, but no insight into when that will be ready.